teorija.tex

```diff
@@ -3810,6 +3810,7 @@ \subsection{Trikotnik}
   p &= \frac{a \krat v_a}{2} = \frac{b \krat v_b}{2} = \frac{c \krat v_c}{2} \\
   p &= \frac{a \krat b \krat \sin\gamma}{2} = \frac{a \krat c \krat\sin\beta}{2} =
   \frac{b \krat c \krat\sin\alpha}{2} \\
+  p &= \sqrt{s\left( s-a \right)\left( s-b \right)\left( s-c \right)} \comment{Heronov obrazec, ki je razložen kasneje v sekciji \ref{sec:liki:heron}}\\
```

Comment:
Skozi cel dokument se uporablja "razdelek", tako da lahko tudi to uporabiš tukaj.